extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -36,16 +40,27 @@ export class NetCoreTool {
 
 	private static _outputChannel: vscode.OutputChannel = vscode.window.createOutputChannel(projectsOutputChannel);
 
+	private netCoreInstallationPresent: boolean | undefined;
```

Comment:
Why do you need another property when you already have `netCoreInstallationPresent`?

Reply:
I used the new one to store the value, so that the new code that looks at "isNetCoreInstallationPresent" property- doesn't have to go through the whole processing again. Is there a better way to do it?

Reply:
Are you ok with this one not necessarily being accurate? How often is findOrInstallNetCore called? It just seems like the small overhead of checking whether a file exists is worth not having to try and keep the state in sync here - but if you expect to query this property a whole bunch then it might be worth it.

Reply:
Got rid of the newly added property.